pitch.md

```diff
@@ -13,14 +14,16 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+Very often in school there is group work. In a subset of those times groups will split off, pick something to work on together and delve into a larger, more isolated project. Group make up is very important in these larger projects as a good/bad group could drastically alter the experience for everyone involved. If students that have similar interests and similar goals then a group could potentially create a great project that everyone involed is proud of, but if a group is made of different interests and some members are not as engaged then the project could end up with a fractured result potentially produced by some subsection of the group. The need met by my idea is the need to create groups that can consistently achieve the aforementioned positive result, and I am helping teachers by saving them time and students by protecting them from the aforementioned negative outcome. Currently teachers often will make these groups themselves, allow students to choose their groups, or randomly assign groups. The first possibility often works out, but requires significant work for the teacher and is infeasible for large class sizes. The second option can be fine sometimes but more often groups that are made up of friends (instead of groups chosen by interests and compatibility) can devolve. Finally, random groupings have a tendency to leave some groups in a good spot, some in an ok spot, and some pretty screwed. Thats not great, and i think it can be better. Imagine if every time there was a group project students could fill out a survey of keyword interests, discuss the amount of work they are interested in committing to the project (in my opinion if there is a group that is all ok with doing B work they should work together), and indicate whether they have interest in being a leader of the group and then the teacher could write some simple code in Grouper and out comes groups that are designed to suit every student's goals. Sounds much better to me.
```

Comment:
So clearly, there are situations when you have certain constraints like "A PM on every team," which are straightforward enough.  But you can also have less obvious questions like "Should we try to even out or match up ability levels/dedication?" 

Do you plan on (eventually) dealing with that as well in some way, or are you putting that beyond the scope of the project?